workshop/Lab0/README.md

```diff
@@ -0,0 +1,114 @@
+
+2. Install Service Catalog by running 
+```
+helm install svc-cat/catalog \
+--name catalog --namespace catalog
+```
+
+You should be able to see the helm tiller and service catalog pods like so:
+```
+$ kubectl get pods --all-namespaces
+NAMESPACE     NAME                                                  READY     STATUS    RESTARTS   AGE
+catalog       catalog-catalog-apiserver-6bc6bb895d-f2xhp            2/2       Running   0          8m
+catalog       catalog-catalog-controller-manager-5c7475f7d4-fwlfg   1/1       Running   1          8m
+kube-system   kube-addon-manager-minikube                           1/1       Running   0          8m
+kube-system   kube-dns-54cccfbdf8-vpvmn                             3/3       Running   0          8m
+kube-system   kubernetes-dashboard-77d8b98585-sw5gc                 1/1       Running   0          8m
+kube-system   storage-provisioner                                   1/1       Running   0          8m
+kube-system   tiller-deploy-865dd6c794-88njn                        1/1       Running   0          9m
+```
+
+# Install svcat
+
+svcat is a command line tool for interacting with Service Catalog. It can be run as a kubectl plugin, or as a standalone binary.
+Download the version that corresponds to your operating system:
+
+* [Linux](https://download.svcat.sh/cli/latest/linux/amd64/svcat)
+* [OS X](https://download.svcat.sh/cli/latest/darwin/amd64/svcat)
+* [Windows](https://download.svcat.sh/cli/latest/windows/amd64/svcat.exe)
+
+Make the binary executable:
+
+* (Linux/OS X) `chmod +x svcat`
+* (Windows) Make sure the filename ends in .exe
+
+Move the file somewhere on your PATH:
+
+* (Linux/OS X) `mv svcat /usr/local/bin/`
+* (Windows) `mkdir -f ~\bin
+$env:PATH += ";${pwd}\bin"`
+
+Ensure that you can see svcat and it can access your Kubernetes deployment:
+```
+$ svcat get classes
+  NAME   DESCRIPTION   UUID  
++------+-------------+------+
+```
+
+# Download the Workshop Source Code
+
+The 'guestbook' repository has the application that we'll be deploying. 
+While we're not going to build it we will use the deployment configuration files from that repo. 
+Guestbook application has two versions v1 and v2 which we will use to demonstrate some rollout 
+functionality later. All the configuration files we use are under the directory guestbook/v1.
+
+The 'svccat' repository contains the step by step instructions to run the workshop.
+
+```
+$ git clone https://github.com/IBM/guestbook.git
+$ git clone https://github.com/IBM/svccat.git
+```
+
+# Create a Private Volume
+
+The service broker we will be deploying as part of this workshop will deploy resources into your Kubernetes cluster. In order
+for these services to have a place to store data, we need to create a Persistant Volume for them to mount. Find the `pv.yaml`
+file in the Lab0 directory of this repository, and then run 
+```
+kubectl create -f pv.yaml`
+```
+If we were using an actual service broker, we wouldn't have to do this step.
```

Comment:
not sure what you mean by "actual" - do you mean a broker managed by someone else?